src/services/getErc20Info.ts

```diff
@@ -0,0 +1,30 @@
+import { Erc20Api } from 'types'
+import { log } from 'utils'
+
+interface Params {
+  address: string
+  erc20Api: Erc20Api
+}
+
+interface Erc20Info {
+  symbol: string
```

Comment:
this should be also optional, right?

Reply:
From ERC20 point of view yes, but we already make it required together with `decimals` https://github.com/gnosis/dex-react/blob/develop/src/types/index.ts#L19-L20.

Should both be made optional again?

Reply:
ups. This is why it was optional at the begining. I know there was a discussion, but I didn't realized we made it mandatory.

I would understand a derived "fake" `label` prop or a `getLabel` function that takes a `TokenDetails` as an argument. But I think it's a mistake to make symbol mandatory

Reply:
Symbol is optional again.